src/templates/loading-block/index.vue

```diff
@@ -7,7 +7,7 @@
     <ProgressBar
       class="sticky"
       mode="indeterminate"
-      style="height: 0.375rem"
+      style="height: 0.375rem; border-radius: 0px"
```

Comment:
@gablisb0a , @cesaroeduardo we really need this? It's better avoid to put style value directly in the component and use the tailwind classes instead.

Reply:
Esse exemplo resolve:
`<div class="h-1/4 rounded-none"></div>`

Reply:
@csfeijo sure, i will update this part of the code.